cumulusci/tasks/salesforce/org_settings.py

```diff
@@ -74,33 +69,106 @@ def capitalize(s):
 
 
 @contextlib.contextmanager
-def build_settings_package(settings: dict, api_version: str):
+def build_settings_package(
+    settings: Optional[dict], object_settings: Optional[dict], api_version: str
+):
     with temporary_dir() as path:
-        os.mkdir("settings")
-        for section, section_settings in settings.items():
-            settings_name = capitalize(section)
-            if section == "orgPreferenceSettings":
-                values = "    " + "\n    ".join(
-                    line
-                    for line in "\n".join(
-                        ORGPREF.format(name=capitalize(k), value=v)
-                        for k, v in section_settings.items()
-                    ).splitlines()
+        if settings:
+            os.mkdir("settings")
```

Comment:
I know we typically try to use pathlib over os module for stuff like this. With an `from pathlib import Path` above, this would be:
```suggestion
            Path.mkdir("settings")
```